dev-env/lib/dade-common

```diff
@@ -108,7 +108,7 @@ buildTool() {
     errcho "Building tools.${attr}${forced}..."
     # Allow to fail, so we can capture outpath and to capture the exit code too.
     set +e
-    outpath=$(nix-build "${DADE_BASE_ROOT}/nix/default.nix" -A tools.$attr -Q -o "${target}")
+    outpath=$(nix-build --no-out-link "${DADE_BASE_ROOT}/nix/default.nix" -A tools.$attr -Q -o "${target}")
```

Comment:
We really want the out link here that’s why we set it via `-o`.

Reply:
So if that's not the one creating all those `result-x` I have, what did? 🤔

Reply:
People running `nix-build nix -A tools -A cached` manually would be my guess :slightly_smiling_face:

Reply:
I definitely have (had) more `result-` symlinks than the number of times I've ran that command... Except, I guess, for the fact that I usually run it in a loop. I'll keep an eye on this. I guess for the time being this PR should be closed.

Reply:
Running it just once generates 83 symlinks for me.

Reply:
You get one for each nix derivation. `tools` and `cached` are sets that each contain a whole bunch of them so you end up with quite a few symlinks.